src/genie/libs/parser/iosxe/tests/ShowAvcSdServiceInfoSummary/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,13 @@
+expected_output = {
+    "active_controller": {
+        "ipv4_address": "10.11.236.21",
```

Comment:
better to use `ip` instead of `ipv4_address` based on output. we may see ipv6 address here.

Reply:
Updated